Samples/ExampleHost/TinyIoC.cs

```diff
@@ -897,7 +897,7 @@ static AppDomain()
 						try
 						{
 							var asm = Assembly.Load(name);
-							assemblies.Add(asm);
+							assemblies.Record(asm);

```

Comment:
The new ProjectReference format doesn't yet support content-only NuGet packages. It's now a bug.